### PR TITLE
Stylesheet comments parsed incorrectly

### PR DESCRIPTION
**Pull request by zakybilfagih**
#441

### CODE REVIEW

**Comment by zakybilfagih** [comment]
Setting skip_whitespace on parse_stylesheet seems to fix the issue

**Comment by davesnx** [comment]
That's a nice fix, by default we were parsing only one style rule, so I asumed skipping whitespaces was needed. No it isn't anymore 👍🏼

File: e2e/melange/src/ui/ui.re [hunk 1]
```diff
@@ -1,10 +1,22 @@
...
+    background-color: green; /* another one */
   }
 
   @media (min-width: 400px) {
+    /* comment 1 */
     div {
+      padding: 10px;
+/** comment
+ multiline 
+*/
```
Comment (davesnx) [inline e2e/melange/src/ui/ui.re:19]: Would you mind adding this as a cram test?

**Review by davesnx — APPROVED** [review]

File: e2e/melange/src/ui/ui.re (hunk 1 above)
Comment (zakybilfagih) [reply e2e/melange/src/ui/ui.re:19]: sure!

**Comment by zakybilfagih** [comment]
Found another problem, seems like a bug with `skip_whitespace`
tez123

**Pull request merged**